schema/articles.js

```diff
@@ -18,7 +18,7 @@ const Articles = {
     },
     sort: ArticleSorts,
   },
-  resolve: (root, options, response, request, { rootValue: { articlesLoader } }) => {
+  resolve: (root, options, response, { rootValue: { articlesLoader } }) => {
```

Comment:
Ugh! That's what I get for not copy/pasting. But question: shouldn't this be:

```js
resolve: (root, options, request, { rootValue: { articlesLoader } }) => {
```

I know we don't use that 3rd argument anywhere, but just trying to keep things consistent. Did I get that right?